doc/market-gr/MyData-fiskaltrust-Mappings.md

```diff
@@ -0,0 +1,208 @@
+| `ChargeItemCase.DiscountedVatRate1` | `0x4752_2000_0000_0093` | `MyDataVatCategory.VatRate13` (2) | 13% | Discounted VAT rate 1 |
+| `ChargeItemCase.DiscountedVatRate2` | `0x4752_2000_0000_0053` | `MyDataVatCategory.VatRate6` (3) | 6% | Discounted VAT rate 2 |
+| `ChargeItemCase.SuperReducedVatRate1` | `0x4752_2000_0000_00D3` | `MyDataVatCategory.VatRate17` (4) | 17% | Super reduced VAT rate 1 |
+| `ChargeItemCase.SuperReducedVatRate2` | `0x4752_2000_0000_0073` | `MyDataVatCategory.VatRate9` (5) | 9% | Super reduced VAT rate 2 |
+| `ChargeItemCase.ParkingVatRate` | `0x4752_2000_0000_0033` | `MyDataVatCategory.VatRate4` (6) | 4% | Parking VAT rate |
+| `ChargeItemCase.NotTaxable` | `0x4752_2000_0000_6017` | `MyDataVatCategory.ExcludingVat` (7) | 0% | Not taxable |
+| `ChargeItemCase.ZeroVatRate` | `0x4752_2000_0000_6027` | `MyDataVatCategory.ExcludingVat` (7) | 0% | Zero VAT rate |
+
+## 2. Payment Method Mappings
+
+| fiskaltrust Payment Type | fiskaltrust Hexcode | myDATA Payment Method | Code | Description |
+|--------------------------|-------------------|----------------------|------|-------------|
+| `PayItemCase.UnknownPaymentType` | `0x4752_2000_0000_0000` | `MyDataPaymentMethods.Cash` | 3 | Cash payment |
+| `PayItemCase.CashPayment` | `0x4752_2000_0000_0001` | `MyDataPaymentMethods.Cash` | 3 | Cash payment |
+| `PayItemCase.NonCash` | `0x4752_2000_0000_0002` | `MyDataPaymentMethods.Cash` | 3 | Non-cash payment |
+| `PayItemCase.CrossedCheque` | `0x4752_2000_0000_0003` | `MyDataPaymentMethods.Check` | 4 | Check payment |
+| `PayItemCase.DebitCardPayment` | `0x4752_2000_0000_0004` | `MyDataPaymentMethods.PosEPos` | 7 | POS/E-POS payment |
+| `PayItemCase.CreditCardPayment` | `0x4752_2000_0000_0005` | `MyDataPaymentMethods.PosEPos` | 7 | POS/E-POS payment |
+| `PayItemCase.VoucherPaymentCouponVoucherByMoneyValue` | `0x4752_2000_0000_000F` | `MyDataPaymentMethods.Check` | 4 | Check payment |
+| `PayItemCase.OnlinePayment` | `0x4752_2000_0000_0006` | `MyDataPaymentMethods.WebBanking` | 6 | Web banking payment |
+| `PayItemCase.AccountsReceivable` | `0x4752_2000_0000_0009` | `MyDataPaymentMethods.OnCredit` | 5 | On credit payment |
+| `PayItemCase.SEPATransfer` (IRIS) | `0x4752_2000_0000_0007` | `MyDataPaymentMethods.IrisDirectPayments` | 8 | IRIS direct payments |
+| `PayItemCase.LoyaltyProgramCustomerCardPayment` | `0x4752_2000_0000_0008` | -1 | N/A | Not supported |
+| `PayItemCase.SEPATransfer` (non-IRIS) | `0x4752_2000_0000_0007` | -1 | N/A | Not supported |
+| `PayItemCase.OtherBankTransfer` | `0x4752_2000_0000_000A` | -1 | N/A | Not supported |
+| `PayItemCase.TransferToCashbookVaultOwnerEmployee` | `0x4752_2000_0000_000B` | -1 | N/A | Not supported |
+| `PayItemCase.InternalMaterialConsumption` | `0x4752_2000_0000_000C` | -1 | N/A | Not supported |
+| `PayItemCase.Grant` | `0x4752_2000_0000_000D` | -1 | N/A | Not supported |
+| `PayItemCase.TicketRestaurant` | `0x4752_2000_0000_000E` | -1 | N/A | Not supported |
```

Comment:
It started counting down again with sepa transfer.. should be 9 not 7.
all of the following are also incorrect